lib/modules/manager/flux/index.ts

```diff
@@ -15,6 +16,8 @@ export const defaultConfig = {
   fileMatch: [systemManifestRegex],
 };
 
+export const categories: Category[] = ['cd', 'kubernetes'];
```

Comment:
```suggestion
export const categories: Category[] = ['cd', 'helm',  'kubernetes'];
```
WDYT? flux also supports helm charts

Reply:
If we would add all supported systems that will become quite convoluted. 
e.g. for Terraform it would contain the categories Helm, Docker, Kubernetes, IaC, ...

Reply:
ok